app/models/concerns/eds_subjects.rb

```diff
@@ -0,0 +1,98 @@
+module EdsSubjects
+  def eds_subjects
+    @eds_subjects ||= Subject.from(self['eds_subjects'])
+  end
+
+  def eds_subjects_geographic
+    @eds_subjects_geographic ||= Subject.from(self['eds_subjects_geographic'])
+  end
+
+  def eds_subjects_person
+    @eds_subjects_person ||= Subject.from(self['eds_subjects_person'])
+  end
+
+  def eds_author_supplied_keywords
+    @eds_author_supplied_keywords ||= Subject.from(self['eds_author_supplied_keywords'])
+  end
+
+  # EDS-centric subject with linking capability
+  class Subject
+    attr_reader :terms
+    delegate :present?, to: :terms
+
+    class Term < Struct.new(:fieldCode, :searchTerm, :label)
+      delegate :to_s, to: :label
+
+      def to_html
+        field = case fieldCode
+                when 'SH'
+                  'subject_heading'
+                when 'DE'
+                  'descriptor'
+                when 'SU'
+                  'subject'
+                when 'KW'
+                  'keyword'
+                else
+                  'subject'
+                end
+        # TODO: we don't have a view context for `link_to` or `articles_path`
+        "<a href=\"/articles?q=#{CGI.escape(searchTerm)}&amp;search_field=#{field}\">#{CGI.escapeHTML(label)}</a>"
```

Comment:
I'm not sure how to get the view context here...

Reply:
Hmm, what about using `Rails.application.routes.url_helpers` to at least get the articles path helper. 

I think we can live w/ the html, but hardcoding the path may be a little more risky.

Reply:
Ok, I've added that in 190684a3be375c68ec1e96395372878117fa48c1